apps/proposal/test/proposal.js

```diff
@@ -0,0 +1,156 @@
+const namehash = require('eth-ens-namehash').hash;
+
+const Proposal = artifacts.require("Proposal.sol");
+const { Contributor, getContributorContract } = require("../../contributor/artifacts");
```

Comment:
idk why I have an error here importing other apps contracts?

Reply:
looks to me like `artifacts` might not defined in the contributor/artifacts.js file. 
`artifacts.require('Contributor')` does not work here? because the proposal app does not know about the Contributor.sol?

from where is truffle loading the contracts?

Reply:
I did the same with the Contribution app to access to the Token app contracts, and it worked...
If I understand correctly `artifacts.require('Contributor.sol')` in Contributor app should b able to access that contract inside that app contracts folder... or no ?